mixer/test/perf/check_test.go

```diff
@@ -0,0 +1,371 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package perftests
+
+import (
+	"testing"
+
+	"istio.io/istio/mixer/pkg/perf"
+	"istio.io/istio/mixer/test/spyAdapter"
+)
+
+func Benchmark_Check_1Client_1Call(b *testing.B) {
```

Comment:
It would be great to have a detailed explanation of these benchmarks as comments.

Reply:
I was hoping that the variable name would do the work; I guess I was wrong :-). Comments always go out of sync. Added more details.